packages/devtools/lib/src/timeline/flame_chart_canvas.dart

```diff
@@ -579,8 +596,12 @@ class TimelineGridNode {
   final String timestampText;
 
   static const gridLineWidth = 0.8;
-  static const gridLineColor = Color.fromRGBO(180, 180, 180, 0.5);
+  static const gridLineColor = Color(0xFFCCCCCC);
```

Comment:
use a ThemedColor and figure out a less bright value in the dark theme.

Reply:
done.